File: BACKEND/PickleballCourtBookingSystem/PickleballCourtBookingSystem.Api/DTOs/AddCourtClusterRequest.cs
using PickleballCourtBookingSystem.Core.Entities;

namespace PickleballCourtBookingSystem.Api.DTOs;

public record AddCourtClusterRequest(
    string Name,
    string? Description,
    TimeSpan OpeningTime,
    TimeSpan ClosingTime,
    string City,
    string District,
    string Ward,
    string Street,
    int NumberOfCourts,
    IFormFile? Image
    );

File: AndroidApp/app/src/main/java/com/maxholmes/androidapp/data/dto/request/AddCourtClusterRequest.kt
package com.maxholmes.androidapp.data.dto.request

data class AddCourtClusterRequest(
    val name: String,
    val description: String,
    val openingTime: String,
    val closingTime: String,
    val city: String,
    val district: String,
    val ward: String,
    val street: String,
    val numberOfCourts: Int,
    val image: String?
)
